src/config.cpp

```diff
@@ -842,6 +843,7 @@ void apply_config(std::unordered_map<std::string, std::string> &&vars) {
 
   int_f(vars, "qp", video.qp);
   int_f(vars, "min_threads", video.min_threads);
+  int_f(vars, "preloaded_frames", video.preloaded_frames);
```

Comment:
You need to validate a minimum value of 2, otherwise Sunshine will crash. I `int_between_f` (see the line directly below).